runtime/jcl/common/jclcinit.c

```diff
@@ -632,6 +635,12 @@ initializeRequiredClasses(J9VMThread *vmThread, char* dllName)
 	if (NULL == objectClass) {
 		return 1;
 	}
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+	identityInterfaceClass = vmFuncs->internalFindKnownClass(vmThread, J9VMCONSTANTPOOL_JAVALANGIDENTITYINTERFACE, J9_FINDKNOWNCLASS_FLAG_NON_FATAL);
```

Comment:
Why is this being force loaded earlier than it would be by the default loading of the vmconstantpool classes?

It would be good to add a comment to describe why it's necessary this early (if it is)

Reply:
@tajila Could you give some input?

Reply:
The class needs to be loaded before the first non j.l.Object|interface|value type (`JAVALANGJ9VMINTERNALS` the second class) is loaded. The vmconstantpool classes are loaded later, after a few core classes have already be loaded.

The reason to do it explicitly is report a JVM internal failure should there be a problem loading the class.

That being said it should be loaded as part of loading J9VMINTERNALS.

Reply:
The loading of this class should be triggered by `J9VMINTERNALS`, so perhaps the explicit classload is not needed.

Reply:
@OussamaSaoudi  can you make that change